vac/raw/sds.md

```diff
@@ -206,7 +213,8 @@ in the incoming buffer.
 For each message in the incoming buffer:
 
 * the participant MAY attempt to retrieve missing dependencies from the Store node
-(high-availability cache) or other peers.
+(high-availability cache) or other peers
+using the `retrieval_hint` in the `HistoryEntry`.
```

Comment:
Subtle difference, but it's important that retrieval can happen without using the `retrieval_hint`.

```suggestion
(high-availability cache) or other peers.
It MAY use the application-specific `retrieval_hint` in the `HistoryEntry` to facilitate retrieval.
```

Reply:
Added in latest commit